README.md

```diff
@@ -77,15 +89,20 @@ chmod +rx /usr/local/bin/dgoss
 make build
 ```
 
+<!-- --8<-- [end:install] -->
+
 ## Full Documentation
 
-Documentation is available here: [manual](https://github.com/goss-org/goss/blob/master/docs/manual.md)
+Documentation is available at <https://goss.rocks/>
 
 ## Quick start
 
+<!-- --8<-- [start:quickstart] -->
+
 ### Writing a simple sshd test
 
-An initial set of tests can be derived from the system state by using the [add](https://github.com/goss-org/goss/blob/master/docs/manual.md#add-a---add-system-resource-to-test-suite) or [autoadd](https://github.com/goss-org/goss/blob/master/docs/manual.md#autoadd-aa---auto-add-all-matching-resources-to-test-suite) commands.
+An initial set of tests can be derived from the system state by using the [add](https://goss.rocks/cli/#add)
```

Comment:
You can't on the README and any file included from outside the documentation.
This is why I said that most probably, on the next iteration you will move some part of it in the documentation and have the readme link to the doc.
MkDocs and GitHub-flavored Markdown are not compatible past the base Markdown specifications. 
This means for github markdown files (readme, contributing...) imported in MkDocs:
- there is no relative page handling
- there is no support for advanced formatting
- they might not respond to the exact same linting rules (but I try to have some which makes both the most compatible possible without surprises)

As for the versioning, (cf. the description of this PR), while this is possible and simple with readthedocs, I don't recommend it for a CLI, only for libs (because you will end by having the same documentation at multiple URLs and while you specify so you know the version of the lib you include in a project, most of the time, you just use the cli version bundled in the environment). It is easier to add notes giving the deprecated version of a flag or the "since" version for new features.

Versioning is interesting for libs because programmatic API surface is often bigger and changes breaks more things (as an integrator, API changes are quite common, as a CLI user, it is very rare that I have a breaking change on a CLI)

Reply:
Oh, it's GitHub content again. 🤕  Yes, agree!